test/config-luatest/log_wrapper_test.lua

```diff
@@ -26,6 +26,7 @@ g.test_jsonify_table = function(g)
         exit_code = 0,
         stdout = '',
         stderr = 'foo: {"bar":"baz"}',
+        env = os.environ(),
```

Comment:
AFAIR, it shouldn't affect anything. Can you tell some details, how it changes the environment for the child process?

Reply:
No.

Reply:
So, let's drop these changes.

Reply:
No.

Reply:
PR is not ready actually, don't waste your time on it. I'll let you know when it will be ready if you want review these changes. I suspect GH automatically requested review from you because you are a code owner (it is a test for config). Just ignore for now.

Reply:
It was not marked as a draft, when I received a email with the review request.

It is OK to be in a middle of an investigation -- just say that you're there.

If you can confirm that this change actually has some effect, it is also useful information per se. It breaks my expectations and, so, interesting.

The tone of your responses saddens me, to be honest.

(BTW, refined-github allows to create a draft pull request without opening a usual PR first -- it adds a separate button on the pull request submission form.)